spec/ruby_terraform/commands/apply_spec.rb

```diff
@@ -107,6 +107,21 @@
         var_file: 'some/vars.tfvars')
   end
 
+  it 'adds a var-file option for each supplied var-file' do
+    command = RubyTerraform::Commands::Apply.new(binary: 'terraform')
+
+    expect(Open4).to(
+        receive(:spawn)
+            .with("terraform apply -var-file=some/vars1.tfvars -var-file=some/vars2.tfvars some/configuration", any_args))
+
+    command.execute(
+        directory: 'some/configuration',
+        var_file: [ 
```

Comment:
This also makes me wonder if a nicer interface would be to allow `:var_file` to pass a single var file path, maintaining backwards compatibility and use `:var_files` for the array. What do you think?

Reply:
Potentially this is more confusing but I find it strange to say `var_file` when the value is actually an array. Potentially we could leave the `var_file` attribute undocumented so that future users don't adopt it and there's less confusion.

I think the way it would work is if both `var_file` and `var_files` were provided, the resulting command would include them all.

Reply:
Ok, I agree with you this could be a better way. I will update the PR. Thanks for your quick response!